content/en/xtc-release-notes/v43.md

```diff
@@ -0,0 +1,29 @@
+---
+title: XTC v43
+linkTitle: XTC v43
+
+weight: 15
+type: docs
+
+date: 2022-01-18
```

Comment:
Even though this is supposed to be the release date of the document, I'd set it to the version's release date (in this case: 2021-11-14).